documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,654 @@
+|                                  |    integral?(num)                             |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n1 = n2                       |    n1 = n2                                    |
+|                                  |    n1 == n2                                   |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n1 < n2                       |    n1 < n2                                    |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n1 > n2                       |    n1 > n2                                    |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n1 <= n2                      |    n1 <= n2                                   |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n1 >= n2                      |    n1 >= n2                                   |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n == 0                        |    zero?(n)                                   |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n > 0                         |    positive?(n)                               |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n < 0                         |    negative?(n)                               |
```

Comment:
The above three should show that `==`, `>`, and `<` also work in Dylan. You might want to somehow show that `0 == 0.0` is false in Dylan.